python/src/opendp/context.py

```diff
@@ -702,12 +702,18 @@ def fix(self, d_in: float, d_out: float, output_measure: Optional[Measure] = Non
 
         The discovered parameter is assigned to the param attribute of the returned transformation or measurement.
         """
-        param = binary_search(
-            lambda x: _cast_measure(self.partial(x), output_measure, d_out).check(
-                d_in, d_out
-            ),
-            T=T,
-        )
+        # When the output measure corresponds to approx-DP, only optimize the epsilon parameter.
+        # The delta parameter should be fixed in _cast_measure, and if not, then the search will be impossible here anyways.
+        if output_measure == fixed_smoothed_max_divergence(T=float):
```

Comment:
Just glancing at the docs for `fixed_smoothed_max_divergence`, it doesn't mention approx-DP. Should it? (Not a blocker for this PR, but maybe file an issue from this comment.)

Reply:
I have another PR that refactors the measures: #1768. Will follow-up in work related to that.